src/man/sssd.conf.5.xml

```diff
@@ -2964,6 +2966,16 @@ min_id = 10000
 max_id = 20000
 enumerate = False
 </programlisting>
+	</para>
+	<para>
+            2. The following example shows configuration in IPA AD trust. AD having
```

Comment:
@fidencio I think you want to mention IPA AD trust, where AD has child domain? If it is then what are your suggestions about the text to be included?